vimeo-networking/src/main/java/com/vimeo/networking/model/TextTrack.java

```diff
@@ -0,0 +1,116 @@
+package com.vimeo.networking.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.stag.UseStag;
+
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * Created by Mohit Sarveiya on 6/29/17.
+ */
+@UseStag
+@SuppressWarnings({"unused", "WeakerAccess"})
+public class TextTrack implements Serializable {
+
+    private static final long serialVersionUID = 3019422318347933808L;
+
+    @SerializedName("uri")
+    protected String mUri;
+
+    @SerializedName("active")
+    protected Boolean mActive;
```

Comment:
I know this is a habit from using Kotlin, but maybe we should use a primitive boolean type here